content/en/monitors/guide/division-in-monitor-evaluations.md

```diff
@@ -0,0 +1,211 @@
+
+## `.fill()`
+
+You can apply a `.fill()` function to ensure all time buckets have valid values. For **gauge** metric types, the default interpolation is linear or `.fill(linear)`. For **count** and **rate** type metrics, the default is `.fill(null)`, which disables interpolation.
+
+**Original**: `sum:my_metric.misaligned.groups.rate{env:a} by {timer,env}`
+
+```
+| Timestamp             | timer:1min,env:a    | timer:2min,env:a    |
+| :-------------------- | :------------------ | :------------------ |
+| 2019-03-29 12:00:00   | 1                   | 1                   |
+| 2019-03-29 12:01:00   | 1                   |                     |
+| 2019-03-29 12:02:00   | 1                   | 1                   |
+| 2019-03-29 12:03:00   | 1                   |                     |
+| 2019-03-29 12:04:00   | 1                   | 1                   |
+```
+
+Let's assume that `my_metric.misaligned.groups` is metric type **rate** so there is no interpolation as default. Consider this query:
+
+```
+sum(last_5m):sum:my_metric.misaligned.groups.rate{timer:1min,env:a} / sum:my_metric.misaligned.groups.rate{timer:2min,env:a}
+```
+
+You would see the following
+
+| Path                | Evaluation                          | Result |
+| :------------------ | :---------------------------------- | :----- |
+| `classic_eval_path` | **1/1 + 1/Nan + 1/1 + 1/Nan + 1/1** | 3      |
+
+By adding interpolation, you can ensure that metrics at every time interval
+
+**Modified**: `sum:my_metric.misaligned.groups.rate{env:a} by {timer,env}.fill(last)`
+
+```
+| Timestamp             | timer:1min,env:a    | timer:2min,env:a    |
+| :-------------------- | :------------------ | :------------------ |
+| 2019-03-29 12:00:00   | 1                   | 1                   |
+| 2019-03-29 12:01:00   | 1                   | 1                   |
+| 2019-03-29 12:02:00   | 1                   | 1                   |
+| 2019-03-29 12:03:00   | 1                   | 1                   |
+| 2019-03-29 12:04:00   | 1                   | 1                   |
+```
+
+Modified query:
+
+```
+sum(last_5m):sum:my_metric.misaligned.groups.rate{timer:1min,env:a} / sum:my_metric.misaligned.groups.rate{timer:2min,env:a}.fill(last)
+```
+
+With `.fill(last)` the new result would be:
+
+| Path                | Evaluation                      | Result |
+| :------------------ | :------------------------------ | :----- |
+| `classic_eval_path` | **1/1 + 1/1 + 1/1 + 1/1 + 1/1** | 5      |
+
+## Short Evaluation Windows
```

Comment:
We probably don't want to go into too many implementation details here, but I definitely agree that we should recommend against small monitor windows for error rate queries. Make we can just say something about avoiding discrepancies if one metric is submitted slightly earlier than another?